src/Symfony/Bridge/Twig/Extension/SecurityExtension.php

```diff
@@ -25,10 +26,12 @@
 class SecurityExtension extends AbstractExtension
 {
     private $securityChecker;
+    private $csrfTokenManager;
 
-    public function __construct(AuthorizationCheckerInterface $securityChecker = null)
+    public function __construct(AuthorizationCheckerInterface $securityChecker = null, CsrfTokenManagerInterface $csrfTokenManager = null)
```

Comment:
Adding the CSRF manager adds more deps, would it be better to create a Twig runtime class at this point?

Reply:
#24692 does it also